Added unit tests

minacometa opened the pull request.

romhaw committed:
Added unit tests

carlsteenkamp commented on ElmersAPIServiceTests/CoffeeTests.cs:
```diff
@@ -0,0 +1,47 @@
+    public class CoffeeTests
+    {
+        [TestMethod()]
+        public void ProcessCoffeeValidInput()
```
We could add DataRowAttributes here and pass in the data to test and the expected result as input

minacometa replied:
Added DynamicData instead of DataRow attribute to handle the custom CoffeeSettings object that is the expected result

carlsteenkamp commented on ElmersAPIServiceTests/NumToTextConverterTests.cs:
```diff
@@ -0,0 +1,46 @@
+    public class NumToTextConverterTests
+    {
+        [TestMethod()]
+        public void ConvertNumberToWords_ValidInput()
```
DataRow and pass it the test cases as per the assessment

minacometa replied:
Added DataRow Attributes for test cases defined in the assessment

romhaw committed:
Added DataRow attributes for the Number to Text Converter.
Added DynamicData attribute for the Coffee Test to handle custom CoffeeSettings object as the expected result.
